tools/python/translate.py

```diff
@@ -149,8 +153,13 @@ def usage():
 
   text_to_translate = ' '.join(sys.argv[1:])
 
-  translations = deepl_translate(text_to_translate)
-  google_translations = google_translate(text_to_translate)
+  source_language = 'en'
+  if len(text_to_translate) > 3 and text_to_translate[2] == ':':
+    source_language = text_to_translate[0:2]
+    text_to_translate = text_to_translate[3:].lstrip()
+
+  translations = deepl_translate(text_to_translate, source_language)
```

Comment:
What's the benefits of calling deepl if there is no option to change the engine in the signature?

Reply:
DeepL translations usually have a better quality than Google's.

Reply:
but your code is not using it right?